judo-runtime-core-dispatcher/src/main/java/hu/blackbelt/judo/runtime/core/dispatcher/behaviours/AlwaysRollbackTransactionalBehaviourCall.java

```diff
@@ -33,7 +33,7 @@
 
 @Slf4j
 public abstract class AlwaysRollbackTransactionalBehaviourCall<ID> implements BehaviourCall<ID> {
-    private static final String ROLLBACK_KEY = "ROLLBACK";
+    public static final String ROLLBACK_KEY = "ROLLBACK";
```

Comment:
Can this be `protected` instead?

Reply:
DefaultDispatcher uses this static field which would not have access to it if it were protected.